RequestManagementSystem/Client/Request.py

```diff
@@ -82,6 +82,7 @@ def __init__( self, fromDict = None ):
     self.RequestName = None
     self.OwnerGroup = None
     self.SourceComponent = None
+    self.RequestID = None
```

Comment:
No no no no. That will break everything. Read the comment: https://github.com/DIRACGrid/DIRAC/blob/integration/RequestManagementSystem/Client/Request.py#L50

Reply:
Sorry, I did this because Travis was failing will remove it.

Reply:
Travis can't be failing because of that, there must be something else

Reply:
@chaen This what I get when I do the `pylint` check `E:477,28: Instance of 'Request' has no 'RequestID' member (no-member)` since this is an error that's why Travis fails.

Reply:
Look how it is done somewhere else, for example https://github.com/DIRACGrid/DIRAC/blob/integration/RequestManagementSystem/Service/ReqManagerHandler.py#L120

Reply:
Thanks this worked.